Gemfile

```diff
@@ -31,6 +31,9 @@ gem "rake", "10.1.1", :require => false
 # Hiera has an unbound dependency on json_pure
 # json_pure 2.0.2+ officially requires Ruby >= 2.0, but should have specified that in 2.0
 gem 'json_pure', '~> 1.8', :require => false
+# i18n support (gettext-setup and dependencies)
+gem 'gettext-setup', '>= 0.8', '< 1.0', :require => false
```

Comment:
Since `0.10` contains support for the `exclude_files` configuration option we added, I think we should update dependencies to reflect `0.10` minimum.